backend/model/top_container.rb

```diff
@@ -265,6 +265,50 @@ def self.batch_update(ids, fields)
   end
 
 
+  def self.bulk_update_container_profile(ids, container_profile_uri)
```

Comment:
thanks mark - lovely.
i made a minor tweak to ensure topcontainers are touched when there is no new container profile